### PR TITLE
Clarify the logical (organization) structure

### PR DESCRIPTION
**Pull request by matgnt**
Relevant in the presentation and issuance flow.
This should make clear that, e.g. Credential Service is NOT a central system.

### CODE REVIEW

File: specifications/M1/issuance.flow.puml [hunk 1]
```diff
@@ -5,10 +5,15 @@
...
 
+box "Client Organization"
 participant "Client" as C
-participant "Client \nCredential Service" as CS
-participant "Client \nSecure Token Service" as STS
+participant "Credential Service" as CS
+participant "Secure Token Service" as STS
+end box
+
+box "Issuer Organization"
```
Comment (paullatzelsperger) [inline specifications/M1/issuance.flow.puml:14]: Just a thought: using the word "Organization" here could mislead readers to think, that the "issuer" is _just another company_, where in actuality it's likely a (or one of several) specially equipped entities.
Maybe "Issuing Entity"? "Issuer Entity"?

Comment (matgnt) [reply specifications/M1/issuance.flow.puml:14]: I think it's still an organization, but fine for me. Same for the 'Client' or stick with 'Organization' there?
Will change it to 'Issuer Entity'.

Comment (matgnt) [reply specifications/M1/issuance.flow.puml:14]: I think we also need to update to 'Client Entity'. Otherwise, we need to explain the difference.

Comment (paullatzelsperger) [reply specifications/M1/issuance.flow.puml]: My point was to distinguish between the "Organisation", i.e. the participant, which can exist many times in a data space, and he issuer, which exists once, or only a few times. 

Just so there is a distinction between these two.

Comment (jimmarino) [reply specifications/M1/issuance.flow.puml]: Issuer Entity is fine for me

**Pull request merged**